.ci/bump-release-version.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+RELEASE_VERSION=$1
+echo RELEASE_VERSION=$RELEASE_VERSION
```

Comment:
lets name script "bump-version.sh"
and variables RELEASE_VERSION -> VERSION

Reason: version update might not be related to release. bump can be done at anytime and release can be done few weeks after bump.

Reply:
Changes Done